src/components/typography/typography.test.tsx

```diff
@@ -0,0 +1,316 @@
+    lineHeight: "20px",
+    fontWeight: "400",
+    textTransform: "none",
+    textDecoration: "none",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when variant is 'big'", () => {
+  render(<Typography variant="big">Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    fontSize: "16px",
+    lineHeight: "24px",
+    fontWeight: "400",
+    textTransform: "none",
+    textDecoration: "none",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when variant is 'sup'", () => {
+  render(<Typography variant="sup">Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    fontSize: "13px",
+    lineHeight: "20px",
+    fontWeight: "400",
+    textTransform: "none",
+    textDecoration: "none",
+    verticalAlign: "super",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when variant is 'sub'", () => {
+  render(<Typography variant="sub">Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    fontSize: "13px",
+    lineHeight: "20px",
+    fontWeight: "400",
+    textTransform: "none",
+    textDecoration: "none",
+    verticalAlign: "sub",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when variant is 'strong'", () => {
+  render(<Typography variant="strong">Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    fontSize: "14px",
+    lineHeight: "21px",
+    fontWeight: "700",
+    textTransform: "none",
+    textDecoration: "none",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when variant is 'b'", () => {
+  render(<Typography variant="b">Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    fontSize: "14px",
+    lineHeight: "21px",
+    fontWeight: "700",
+    textTransform: "none",
+    textDecoration: "none",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when variant is 'em'", () => {
+  render(<Typography variant="em">Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    fontSize: "14px",
+    lineHeight: "21px",
+    fontWeight: "700",
+    textTransform: "none",
+    textDecoration: "underline",
+    margin: "0",
+  });
+});
+
+test("should render with expected styles when 'truncate' is true", () => {
+  render(<Typography truncate>Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+  });
+});
+
+test("should render with visuallyHidden styles when screenReaderOnly is true", () => {
+  render(<Typography screenReaderOnly>Test</Typography>);
+
+  expect(screen.getByText("Test")).toHaveStyle({
+    border: "0",
+    height: "1px",
+    margin: "-1px",
+    overflow: "hidden",
+    padding: "0",
+    position: "absolute",
+    width: "1px",
+    whiteSpace: "nowrap",
+  });
+});
+
+test("should override 'display' property when passed", () => {
```

Comment:
**question**: was there any particular reason why this was included from the override tests, coverage?

Reply:
Yeah this one was for coverage